lib/module.js

```diff
@@ -92,5 +96,6 @@ const extendWithSassResourcesLoader = matchRegex => resources => (config) => {
 const extendSass = extendWithSassResourcesLoader(/sass/)
 const extendScss = extendWithSassResourcesLoader(/scss/)
 const extendLess = extendWithSassResourcesLoader(/less/)
+const extendPostCSS = extendWithSassResourcesLoader(/\?css/)
```

Comment:
Nuxt uses a loader test that looks like this: `/\.p(ost)?css$/i`, so I test for the `?css` portion to match this, but ignore `.css` files.